### ISSUE TITLE
CSS in the WooCommerce addon is being enqueued incorrectly

### ISSUE THREAD

**Issue by VR51** [strangerstudios/pmpro-woocommerce#53]
The CSS in the WooCommerce addon is being enqueued incorrectly. Easy enough to fix.

<strong>Edit file:</strong>

```
pmpro-woocommerce/pmpro-woocommerce.php
```

<strong>Replace:</strong>

```
/**
 * Enqueue CSS
 */
function pmprowoo_enqueue_css() {
	wp_register_style( 'pmprowoo', plugins_url(__FILE__, '/css/style.css' ), null );
	wp_enqueue_style('pmprowoo');
}
```

<strong>With:</strong>

```
/**
 * Enqueue CSS
 */
function pmprowoo_enqueue_css() {
	wp_register_style( 'pmprowoo', plugins_url( '/css/style.css', __FILE__ ), null );
	wp_enqueue_style('pmprowoo');
}
```
The __FILE__ variable is in the wrong position.

**Comment by strangerstudios** [comment]
Fixed in the 1.4.2 update pushed out. Please open a new issue if you are still having related problems.

**Comment by strangerstudios** [comment]
p.s. Thanks!
